### ISSUE TITLE
🚀 Add chromatic tests for 120% zoom.

### ISSUE THREAD

**Issue by patricklafrance** [gsoft-inc/sg-orbit#721]
## Description

We should tests that every components renders correctly when a zoom is applied.

Chromatic doesn't have support for zoom but we could wrap the component in a decorator that applies a css transform scale(120%).

_CSS zoom might not be 1:1 with the browser zoom so your mileage may vary.

**Comment by patricklafrance** [comment]
@fraincs this kind of chromatic tests but be relevant to validate the components responsive support.

**Comment by fraincs** [comment]
You mean we should add responsive tests to components in chromatic? what about zoom? I think it would not represent the truth as zooming is different from a css transform Scale.

**Comment by patricklafrance** [comment]
@fraincs yup, maybe not a full coverage since it would be $$$ but at least a few tests per components.

Would love to use a native zoom feature but it's not supported by chromatic. If you think transform scale wouldn't help at all, then let's not do it :)

**Comment by fraincs** [comment]
as for zoom css `zoom: 200%` is a better avenue

**Comment by patricklafrance** [comment]
Oh, we could use https://www.chromatic.com/docs/viewports for responsive test.

**Comment by patricklafrance** [comment]
> as for zoom css `zoom: 120%` is a better avenue it still would be expensive to test zooming

Oh, pretty cool, does it really match a browser zoom in ?

**Comment by fraincs** [comment]
> > as for zoom css `zoom: 120%` is a better avenue it still would be expensive to test zooming
> 
> Oh, pretty cool, does it really match a browser zoom in ?

yes
